<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PackagesPage</class>
 <widget class="QWidget" name="PackagesPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>723</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Packages</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Tab 1</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="ModListView" name="modTreeView">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="acceptDrops">
          <bool>true</bool>
         </property>
         <property name="dragDropMode">
          <enum>QAbstractItemView::DropOnly</enum>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="install">
           <property name="text">
            <string>&amp;Install</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="changeVersion">
           <property name="text">
            <string>Change &amp;Version</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="revert">
           <property name="text">
            <string>Revert</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="remove">
           <property name="text">
            <string>&amp;Remove</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QProgressBar" name="progressBar">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="format">
            <string>Verifying</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="applyTransaction">
           <property name="text">
            <string>Apply</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="revertTransaction">
           <property name="text">
            <string>Revert</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ModListView</class>
   <extends>QTreeView</extends>
   <header>gui/widgets/ModListView.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>modTreeView</tabstop>
  <tabstop>changeVersion</tabstop>
  <tabstop>revert</tabstop>
  <tabstop>remove</tabstop>
  <tabstop>applyTransaction</tabstop>
  <tabstop>revertTransaction</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
